apps/core/scenes/bookingDetail/menuGroups/Passengers/__tests__/__snapshots__/Bag.test.js.snap

```diff
@@ -22,13 +22,12 @@ Object {
         name="baggage-checked"
         size="small"
       />
-      <Text
-        expo={false}
+      <WithTheme
```

Comment:
Snapshot renders `withTheme`?! Weird!

Reply:
True. But I think that main problem is basically in our usage of shallow rendering with combination of `react-native-testing-library` implementation.
It seems like `enzyme` have the similar issue https://github.com/airbnb/enzyme/issues/539 but sadly there doesn't seem to be any equivalent to suggested `.first().shallow()` inside `react-native-testing-library`.

As we mentioned before, we should probably rethink and unify out testing strategy https://kentcdodds.com/blog/why-i-never-use-shallow-rendering .